docs/diagrams/choonwei/DeleteRecurringSessionActivityDiagram.puml

```diff
@@ -0,0 +1,32 @@
+@startuml
+start
+:User executes delete recurring session command;
+
+'Since the beta syntax does not support placing the condition outside the
+'diamond we place it as the true branch instead.
+
+if () then ([all prefixes are present])
+    : Parse all student's arguments;
```

Comment:
Maybe using session instead of student here would be better.

Reply:
Nice catch! Updated!